pallets/ethereum-xcm/src/lib.rs

```diff
@@ -104,10 +105,14 @@ pub mod pallet {
 	}
 
 	#[pallet::pallet]
-	#[pallet::generate_store(pub(super) trait Store)]
 	#[pallet::without_storage_info]
 	pub struct Pallet<T>(PhantomData<T>);
 
+	/// Global nonce used for building Ethereum transaction payload.
+	#[pallet::storage]
+	#[pallet::getter(fn nonce)]
+	pub(crate) type Nonce<T: Config> = StorageValue<_, U256, ValueQuery>;
```

Comment:
We probably need the `OnEmpty` trait to be filled to avoid a migration (and retrieve a 0) https://github.com/paritytech/substrate/blob/963ee8a37130f6d76b2859e76164a01607a306cf/frame/support/src/storage/types/value.rs#L39